tests/e2e/scale/test_scale_small_file_workload.py

```diff
@@ -0,0 +1,110 @@
+"""
+Test to exercise Small File Workload
+
+"""
+
+# Builtin modules
+import logging
+import time
+
+# 3ed party modules
+import pytest
+
+# Local modules
+import ocs_ci.ocs.resources.pod as pod
+from ocs_ci.ocs.ripsaw import RipSaw
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, scale
+from ocs_ci.ocs.small_file_workload import smallfile_workload
+
+log = logging.getLogger(__name__)
+
+
+def get_cephfs_data():
+    """
+    Look through ceph pods and find space usage on all cephfilesystem pods
+
+    Returns:
+        Dictionary of byte usage, indexed by pod name.
+    """
+    ceph_pod = pod.get_ceph_tools_pod()
+    ceph_status = ceph_pod.exec_ceph_cmd(ceph_cmd="ceph df")
+    ret_value = {}
+    for pool in ceph_status["pools"]:
+        if "cephfilesystem" in pool["name"]:
+            ret_value[pool["name"]] = pool["stats"]["bytes_used"]
+    return ret_value
+
+
+@pytest.fixture(scope="module")
+def scale_leaks(request):
+    """
+    For scale tests, get bytes used by cephfs at the start,
+    and compare with the cephfs usage at the finish.
+    """
+    pytest.ceph_fs_use = get_cephfs_data()
+
+    def teardown():
+        """
+        After all the ripsaw creation and deltion runs have finished,
+        test to see if there was a leak in storage.
+        """
+        log.info("In scale_leaks teardown")
+        time.sleep(300)
```

Comment:
why have you defined a hard sleep for 300 seconds?

Reply:
We don't need this.  I removed it and I will rerun the jenkins test.